src/main/resources/avro/AnotherUser.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "record",
+  "namespace": "org.example",
+  "name": "AnotherUser",
```

Comment:
You have not introduced Avro schema versioning based on a version field. That is one of the requirements for this task to include that field.